### PR TITLE
Update project to be installed and released via poetry

### PR DESCRIPTION
**Pull request by oharan2**
- Verified VERSION file
- Updated project metadata and dependencies
- Updated `poetry.lock` according new changes
- Update README

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,10 +1,35 @@
+[tool.black]
+line-length = 88
+target_version = ['py37', 'py38', 'py39', 'py310', "py311"]
```
Comment (rnetser) [inline pyproject.toml:3]: please remove `py37`

**Review by rnetser — CHANGES_REQUESTED** [review]
Please update PR title

**Review by myakove — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Pull request merged**